caravel/assets/spec/javascripts/explore/components/QueryAndSaveBtns_spec.jsx

```diff
@@ -0,0 +1,37 @@
+var React = require('react');
```

Comment:
just curious what's the advantage of using `var` for packages vs `const` ? or `import`ing them?